lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -34,6 +34,28 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Take this with you:")}</h3>
 
+                    <a href="" class="action action-share-facebook btn btn-overlay btn-small icon-only" id="action-share-facebook">
+                        <i class="icon fa fa-facebook-official" aria-hidden="true"></i>
+                        <span class="action-label">${_("Post this on Facebook")}</span>
+                    </a>
+
+                    <a href="" class="action action-share-twitter btn btn-overlay btn-small icon-only" id="action-share-twitter">
+                        <i class="icon fa fa-twitter" aria-hidden="true"></i>
+                        <span class="action-label">${_("Tweet this Accomplishment")}</span>
+                    </a>
+
+                    <a href="" class="action action-share-twitter btn btn-overlay btn-small icon-only" id="action-share-twitter">
+                        <i class="icon fa fa-linkedin" aria-hidden="true"></i>
+                        <span class="action-label">${_("Add to LinkedIn Profile")}</span>
```

Comment:
Ahh that's a quick dupe I made for myself for local testing (since I don't have LinkedIn logic configured locally). Removing now.Thanks for the catch!